plugins/woocommerce-blocks/docs/third-party-developers/extensibility/checkout-block/available-filters/coupons.md

```diff
@@ -7,7 +7,7 @@ The following Coupon filters are available:
 -   [`showRemoveCouponNotice`](#showremovecouponnotice)
 
 ## `coupons`
-
+<!-- markdownlint-disable MD024 -->
```

Comment:
I see that for other files, we're placing the lint rule below the docs header and above the docs title. For consistency-reasons, I suggest keeping the same format across all files.